### PR TITLE
Tipping Banner accessibility/style/copy updates

### PR DESCRIPTION
**Pull request by ryanml**
Fixes: https://github.com/brave/brave-browser/issues/6147

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- [x] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [x] Ran `git rebase master` (if needed).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [ ] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Test Plan:
Defined in issue, be sure to have BAT on hand

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: components/brave_rewards/resources/ui/components/donate/index.tsx [hunk 1]
```diff
@@ -45,13 +45,29 @@ export interface Props {
...
+    }
+    this.sendButton = null
+  }
+
+  componentDidMount () {
+    if (this.sendButton) {
+      this.sendButton.focus()
+    }
+
+    const selectedAmount: HTMLElement | null = document.querySelector('[data-selected="selected"]')
```
Comment (ryanml) [inline components/brave_rewards/resources/ui/components/donate/index.tsx]: I was not able to make a dynamic node reference for the initial selected amount. It needs to validate so the "not enough funds" message shows up if the user has zero bat/less than the initial select.

**Comment by ryanml** [comment]
@NejcZdovc per discussion querySelector call was removed, validation happens at the `Amount` component level, still couldn't nail down a dynamic ref

**Comment by masparrow** [comment]
@ryanml Changes LGTM, but FYI; 

- Clean install
- Enable rewards
- http://vimeo.com/willchristiansen
- Autoplay is disabled by default (Version 0.72.52 Chromium: 77.0.3865.90 (Developer Build) (64-bit))
- this causes ‘Page unresponsive’ popups which interferes with tip panel - this is just an FYI with QA in mind

But, with Autoplay enabled;
- Send tip is focused, but tabbing takes you out of the ‘Tip amount’ pane to the URL, and then tabbing again just goes ‘somewhere’ in the main page behind the Tip panel
- I’d expect tab to be restricted within the ‘Tip amount’ pane so that; Send Tip -> 1.0 BAT -> 5.0 BAT -> 10.0 BAT -> Make this monthly -> Send Tip -> 1.0 BAT and so on.

**Comment by ryanml** [comment]
rebased

File: components/brave_rewards/resources/ui/components/donate/index.tsx [hunk 2]
```diff
@@ -124,12 +147,15 @@ export default class Donate extends React.PureComponent<Props, State> {
           <StyledDonationTitle isMobile={isMobile}>{title}</StyledDonationTitle>
             <StyledAmountsWrapper isMobile={isMobile}>
               {
-                donationAmounts && donationAmounts.map((donation: Donation) => {
+                donationAmounts && donationAmounts.map((donation: Donation, index: number) => {
+                  const isCurrentAmount = donation.tokens === currentAmount.toString()
+                  const isDefaultAmount = index === 0 && !this.state.amountChanged
```
Comment (NejcZdovc) [inline components/brave_rewards/resources/ui/components/donate/index.tsx]: I talked with @davidtemkin and he is ok with going with a middle selection. So let's change index here from 0 to 1.

Comment (ryanml) [reply components/brave_rewards/resources/ui/components/donate/index.tsx]: done

**Pull request merged**